docs/WalletGuide.md

```diff
@@ -40,3 +48,115 @@ val walletCustom = Wallet(StellarConfiguration.Testnet, ApplicationConfiguration
 <!--- INCLUDE
 val wallet = Wallet(StellarConfiguration.Testnet)
 -->
+
+<!--- INCLUDE .*stellar.*
+import org.stellar.walletsdk.*
+import org.stellar.walletsdk.asset.IssuedAssetId
+import org.stellar.walletsdk.horizon.sign
+
+suspend fun main() { 
+-->
+<!--- SUFFIX .*stellar.*
+}    
+-->
+
+## Build on Stellar
```

Comment:
Can you split account and transaction builder into 2 files?
Also, formatting with 2 spaces in the beginning would also be nice